packages/shylu/shylu_dd/frosch/src/Tools/FROSch_Tools_def.hpp

```diff
@@ -474,6 +473,56 @@ namespace FROSch {
         return MapFactory<LO,GO,NO>::Build(tmpGraph->getRowMap()->lib(),-1,repeatedIndices(),0,graph->getRowMap()->getComm());
     }
 
+    template <class LO,class GO,class NO>
+    RCP<const Map<LO,GO,NO> > BuildRepeatedMapOld(RCP<const CrsGraph<LO,GO,NO> > graph)
```

Comment:
If it's "Old", why not get rid of it?  If you can't because some application is using it, consider marking it with a deprecation macro.  We can show you how to set that up if it doesn't already exist in ShyLU.